certified-connectors/Mural/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "[[ 00000000-00000000-00000000-00000000]]",
```

Comment:
@dmytroIvanovDev Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
"clientId": "[[DUMMY]]"

Reply:
@dmytroIvanovDev This needs to be addressed and resolved